docs/source/guides/dm.md

```diff
@@ -83,3 +83,331 @@ has more support to integrate various capabilities of the DM tools.
 
 For more information about DM, see its [API
 Reference](https://git.aps.anl.gov/DM/dm-docs/-/wikis/DM/Beamline-Services/API-Reference).
+
+## Example session
+
+Let's work through an example session at an APS beamline.  The instrument
+package is named `mic_instrument`.  Start a new console session:
```

Comment:
let's make this generalized to our package

Reply:
See commit e52d198c18.